vendors/asurite/asu_directory.php

```diff
@@ -0,0 +1,124 @@
+<?php // this is sourced from the GIOS API
+// This is a static class for interacting with the ASU public directory
+class ASUDirectory {
```

Comment:
This being copy/pasted in isn't my favorite way to do this.. but adding composer just for this file is a bit of work too.  The only reason I bring this up is that these directory endpoints will be going away "soon" and we'll need to figure out equivalent ones using the new iSearch tools.